test/app/teams_test.rb

```diff
@@ -91,6 +91,17 @@ def test_user_must_be_manager
     end
   end
 
+  def test_logged_in_user_can_search_teams
+    [
+      [:get, '/teams'],
+      [:get, '/teams/']
+    ].each do |verb, endpoint|
```

Comment:
> The one reason I have to not use it with minitest is that once you have an assertion, none of the following test cases are run, so it can obscure the scope of an issue.

Agreed 👍 . That's the only problem I see with multiple assertions in a single test. I don't mind it being either ways, but it would be good to have consistency in this file (atleast if not the entire project). If we split up the tests, we should do for others in the file as well - which should be another PR IMO.